centchain/api_integration_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build integration
```

Comment:
New way of defining annotations since go 1.17, I might clean up the older way too